<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cal</class>
 <widget class="QWidget" name="cal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>398</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="txtEventList">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>11</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>392</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
